core/modules/node/tests/node.test

```diff
@@ -1558,31 +1558,31 @@ class NodeSaveTestCase extends BackdropWebTestCase {
 
     $node->save();
     $node = $this->backdropGetNodeByTitle('testing_node_presave', TRUE);
-    $this->assertEqual($node->created, 280299600, 'Saving a node uses "created" timestamp set in presave hook.');
-    $this->assertEqual($node->changed, 979534800, 'Saving a node uses "changed" timestamp set in presave hook.');
+    $this->assertEqual($node->created, 1421394600, 'Saving a node uses "created" timestamp set in presave hook.');
+    $this->assertEqual($node->changed, 1474011660, 'Saving a node uses "changed" timestamp set in presave hook.');
 
     // Programmatically set the timestamps on the node.
     $edit = array(
       'uid' => $this->web_user->uid,
       'type' => 'post',
       'title' => $this->randomName(8),
-      'created' => 280299600, // Sun, 19 Nov 1978 05:00:00 GMT
-      'changed' => 979534800, // Drupal 1.0 release.
+      'created' => 1421394600, // Backdrop v1.0.0 release.
+      'changed' => 1474011660, // Backdrop v1.5.0 release.
     );
 
     entity_create('node', $edit)->save();
     $node = $this->backdropGetNodeByTitle($edit['title']);
-    $this->assertEqual($node->created, 280299600, 'Creating a node uses user-set "created" timestamp.');
-    $this->assertNotEqual($node->changed, 979534800, 'Creating a node doesn\'t use user-set "changed" timestamp.');
+    $this->assertEqual($node->created, 1421394600, 'Creating a node uses user-set "created" timestamp.');
+    $this->assertNotEqual($node->changed, 1474011660, 'Creating a node doesn\'t use user-set "changed" timestamp.');
 
     // Update the timestamps.
-    $node->created = 979534800;
-    $node->changed = 280299600;
+    $node->created = 1474011660;
+    $node->changed = 1421394600;
```

Comment:
Oops... changed is before created. :wink: 

```
> date -d @1474011660
Fr 16. Sep 09:41:00 CEST 2016
> date -d @1421394600
Fr 16. Jan 08:50:00 CET 2015
```
Check the others, too...

Reply:
I don't see how the changes are different from the original...

created: 979534800 = 01/15/2001
changed: 280299600 = 11/19/1978

created: 1474011660 = 09/16/2016
changed: 1421394600 = 01/16/2015

Reply:
I'm just trying to be helpful, and this timestamp thing was the first that caught my eye.

Reply:
You're right, :+1:  this is unrelated. Mark this comment as resolved (I can't).